pkg/crane/append.go

```diff
@@ -47,9 +48,25 @@ func Append(base v1.Image, paths ...string) (v1.Image, error) {
 		return nil, fmt.Errorf("getting base image: %w", err)
 	}
 
+	if base == nil {
```

Comment:
`isWindows` checks for `img==nil` already, we could remove that and put the `== nil` check before `isWindows`.

Reply:
makes sense.

Reply:
fixed it